source/common/common/hash.h

```diff
@@ -135,11 +134,7 @@ class MurmurHash {
   static uint64_t murmurHash2(absl::string_view key, uint64_t seed = STD_HASH_SEED);
 
 private:
-  static inline uint64_t unalignedLoad(const char* p) {
-    uint64_t result;
-    safeMemcpyUnsafeSrc(&result, p);
-    return result;
-  }
+  static inline uint64_t unalignedLoad(const char* p) { return absl::little_endian::Load64(p); }
```

Comment:
`absl::little_endian` is defined in `absl/base/internal/endian.h`. Should we be using something from `internal`? I'm not sure what the API stability of `internal` code is.

Reply:
Thanks for reviewing @ggreenway. I didn't realize  `absl::little_endian` was from an `internal` file. From the preceding comment, I might not end up using it but I'll see if it can be replaced with something equivalent.

Reply:
@ggreenway, I have pushed new changes that use the `htole64` function instead of `absl::little_endian::Load64`